packages/analytics/analytics-chart/src/components/AnalyticsChart.cy.ts

```diff
@@ -312,7 +312,7 @@ describe('<AnalyticsChart />', () => {
     // if you're in between bars.
     cy.get('body').then(($body) => {
       if ($body.find('.tooltip-container').length) {
-        cy.get('.subtitle').should('contain.text', 'service A')
+        cy.get('.subtitle').should('exist')
```

Comment:
It's kind of a "better than nothing" situation for this test case. 

I'm using that pseudo mouse move event to trigger the tooltip, and it works pretty reliably, but there's no guarantee the tooltip will show up because of the nature of it, so the test does the check only if the tooltip actually shows up.

Reply:
That sounds like a recipe for a flaky test: if the subtitle ever doesn't exist, the test will sometimes pass (because the tooltip didn't show up) and sometimes fail (because subtitle wasn't there).  I think we should remove it entirely unless we have a way of guaranteeing that the tooltip shows up.

Reply:
It will never produce false negatives which is what we don't want. 

The threshold that this produces false positives is low enough that I think it is still a valid test. 

If it fails, we know that there's an issue with the tooltip context. I'd like that to still be there, even if there's small chance this test will produce a false positive.

Reply:
The thing I'm concerned about is that this creates a situation where the PR can randomly pass CI, but then fail once it's merged into main.  I think that's a situation that we'd like to avoid.

Is there any way to get rid of the conditional?  Can you calculate where the mouse needs to be in order to show the legend, then use a `cy.should` to wait for it to show up?

Reply:
Ok giving that a try, made some changes to the test